sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -514,31 +547,37 @@ function createZumoNamespace() {
         TSSkipped: TSSkipped,
         AllTestsGroupName: AllTestsGroupName,
         AllTestsUnattendedGroupName: AllTestsUnattendedGroupName,
-        constants: {
+        constants: {
             CLIENT_VERSION_KEY: ClientVersionKey,
             SERVER_VERSION_KEY: ServerVersionKey,
-            RUNTIME_FEATURES_KEY: RuntimeFeaturesKey
+            RUNTIME_FEATURES_KEY: RuntimeFeaturesKey
         },
-        runtimeFeatureNames: {
+        runtimeFeatureNames: {
             STRINGREPLACE: 'stringReplace',
-            DotNetRuntime_Only: 'dotnetRuntimeOnly',
+            DotNetRuntime_Only: 'dotNetRuntimeOnly',
             NodeRuntime_Only: 'nodeRuntimeOnly',
             INT_ID_TABLES: 'intIdTables',
             STRING_ID_TABLES: 'stringIdTables',
-            NH_PUSH_ENABLED: 'nhPushEnabled'
+            NH_PUSH_ENABLED: 'nhPushEnabled'
         },
         Test: ZumoTest,
         Group: ZumoTestGroup,
         tests: {},
-        util: {
+        util: {
             createSeparatorTest: createSeparatorTest,
             compare: compareValues,
             traceResponse: traceResponse,
             dateToString: dateToString,
             randomDate: randomDate,
-            globalTestParams: {}
-        }
-    };
+            globalTestParams: {}
+        },
+        assert: {
+            areEqual: assertAreEqual,
+            areNotEqual: assertAreNotEqual,
+            isTrue: assertIsTrue,
+            isFalse: assertIsFalse
+        }
+    };
```

Comment:
Agree about the assertion library, as you say we may want to revisit the test framework when we have time.